db/schema.rb

```diff
@@ -11,12 +11,13 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160128233820) do
+ActiveRecord::Schema.define(version: 20160129095819) do
 
   create_table "answers", force: :cascade do |t|
     t.integer  "user_id"
     t.integer  "question_id"
     t.string   "content"
+    t.integer  "votes"
```

Comment:
Thought we would not need this again when we created the votes model. Is there anything I'm missing or have missed?